template/en/default/robots.txt.tmpl

```diff
@@ -18,6 +18,8 @@ Disallow: [% basepath FILTER none %]show_bug.cgi*format=multiple*
 
 Allow: [% basepath FILTER none %]describecomponents.cgi
 Allow: [% basepath FILTER none %]describekeywords.cgi
+Disallow: [% basepath FILTER none %]showdependency*
+Disallow: [% basepath FILTER none %]banmyipforever.cgi
```

Comment:
where is this defined?

Reply:
It doesn't exist, it's meant as bait for badly behaved bots.

Reply:
ah, so you can see in the logs when some bots misbehave?